sbirez/management/commands/importtopics.py

```diff
@@ -27,15 +27,15 @@ def handle(self, mdb_filename, solicitation_name, clear=False, **options):
             raise OSError("MS Access database %s not found" % mdb_filename)
 
         # Confirm that solicitation_name is for solicitation in the database
-        proc = subprocess.Popen(#'ls',
```

Comment:
We don't have psql available on the cloud foundry machine.

Instead of removing it, we could wrap that in a try/catch so it wouldn't stop the process OR make it an optional check.